app/src/main/res/layout/video_layout.xml

```diff
@@ -19,4 +20,10 @@
         android:tag="network"
```

Comment:
My gut feel is that XML based widgets want to provide an id, that a list adapter wants to generate an id and that the tag field is used as a payload to provide additional data.  We should talk about this on a case by case basis.

Reply:
I wasn't sure about adding an ID to every TextView in every CardView (and thus in a worse-case-scenario having a dozen or so buttons on the screen all with the same ID) but using an ID makes things easier and I haven't run into any problems with testing yet, so I think it's okay to use an ID instead of a tag.